src/components/Link.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <router-link v-if="this.options.link" :to="this.options.link">
+    <slot />
+  </router-link>
+  <a class="clickable" v-else @click="this.click" >
+    <slot />
+  </a>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator';
+
+export interface LinkOptions {
+  link?: string;
+  onClick?: () => void;
```

Comment:
are there any links without the onClick function?

Reply:
Indeed there are, take a look at the resourceInstancesToList() `utils/Utils.ts`. There is an exemplary empty link object.